README.md

```diff
@@ -1,59 +1,59 @@
+Using any other operating system may result in unexpected installation failures or may break your system, so if not using RPi-OS 32bit, **be prepared for failures.**
 
-## installation files and such
-I have organized everything into folders. Some folders may have install scripts, some may have packages, or links, or something else. Only use this repository if you know what you're doing.
+## Installation Files and Derivatives
+Everything is organized into folders. Some folders may have install scripts, some may have packages and or links or something else. Only use this repository if you know what you're doing, otherwise you may break your installation of your OS.
 
 
 ## To add to [pi-apps](https://github.com/Botspot/pi-apps/):
 
-These are the scripts i'm working on adding to botspot's Pi apps project. If you would like to help me with them, awesome! just message me: @chunkymilk0309#3309 on discord.
+These are the scripts I'm working on adding to Botspot's Pi apps project. If you would like to help me with them, awesome! just message me: @chunkymilk0309#3309 on Discord.
 
 Alacritty https://github.com/chunky-milk/pi-bashscripts-files/tree/main/alacritty
 
 Bleachbit https://github.com/chunky-milk/pi-bashscripts-files/tree/main/bleachbit
 
-conky manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/conky-manager
+Conky Manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/conky-manager
 
-dolphin emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
+Dolphin Emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
 
-dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
+Dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
 
 Duckstation https://github.com/chunky-milk/pi-bashscripts-files/tree/main/duckstation
 
-freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
+Freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
 
-googleearth https://github.com/chunky-milk/pi-bashscripts-files/tree/main/google-earth
+Google Earth https://github.com/chunky-milk/pi-bashscripts-files/tree/main/google-earth
 
-lutris game manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/lutris
+Lutris game manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/lutris
 
-megasync https://github.com/chunky-milk/pi-bashscripts-files/tree/main/megasync
+Megasync https://github.com/chunky-milk/pi-bashscripts-files/tree/main/megasync
 
-obs studio https://github.com/chunky-milk/pi-bashscripts-files/tree/main/obs-studio
+OBS Studio https://github.com/chunky-milk/pi-bashscripts-files/tree/main/obs-studio
 
-openscad https://github.com/chunky-milk/pi-bashscripts-files/tree/main/openscad
+Openscad https://github.com/chunky-milk/pi-bashscripts-files/tree/main/openscad
```

Comment:
there is a issue for OpensCad in the pi-apps repo already (https://github.com/Botspot/pi-apps/issues/556).